apps/condo/domains/miniapp/access/B2BAppPermission.js

```diff
@@ -30,6 +34,26 @@ async function canReadB2BAppPermissions ({ authentication: { item: user }, listK
         return { app: { accessRights_some: { user: { id: user.id }, deletedAt: null } } }
     }
 
+    if (user.type === STAFF) {
+        const userEmployeesWithCanManageRoles = await find('OrganizationEmployee',
+            {
```

Comment:
Why not just using filter from queryEmployee? (to change it later in single place)

Reply:
`queryOrganizationEmployeeFor` not includes `isAccepted` filter (and refactor `queryOrganizationEmployeeFor` not part of this pr). But ok, use `queryOrganizationEmployeeFor` here and add additional filter